documentation/asciidoc/stories/assembly_monitoring.adoc

```diff
@@ -0,0 +1,14 @@
+ifdef::context[:parent-context: {context}]
+[id='prometheus']
+:context: monitor
+= Monitoring {brandname} Services
+{brandname} clusters export statistics and events via a metrics endpoint.
+You can collect these metrics with Prometheus to monitor your {brandname} services.
```

Comment:
"Prometheus can scrape these metrics to allow you to monitor your {brandname} cluster."